### ISSUE TITLE
Add order parameter to toggleSort method to control ascending/descending

### ISSUE THREAD

**Issue by Dobby89** [Buuntu/react-final-table#21]
It would be useful to be able to define which direction you want to sort the column by.

```javascript
toggleSort('columnName') // Sort flip flip
toggleSort('columnName', false) // Sort DESC
toggleSort('columnName', true) // Sort ASC
```

**Comment by Buuntu** [comment]
That's a good idea, I'd accept a PR for this but probably won't get to it soon.

**Comment by Dobby89** [comment]
Cool. Pull request submitted 👍

**Comment by Buuntu** [comment]
Merged in #22
